arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/continuations/Raise.kt

```diff
@@ -211,6 +242,16 @@ public interface Raise<in R> {
   ): A = fold({ catch(it) }, { raise(it) }, { it })
 }
 
+/**
+ * Execute an [action], and map every error of type [E]
+ * into one of type [R].
+ */
+@EffectDSL
+public inline fun <R, E, A> Raise<R>.mapLeft(
```

Comment:
since `mapLeft` is just recover can we remove this? and inline it into `mapOrAccumulate`

Reply:
Done! I've kept `mapLeft` in the code, because it might be useful elsewhere.

Reply:
The method is still there can we add this in a separate PR :)

Reply:
Indeed, I've just removed it to make this PR self-contained